addons/ui/stringtable.xml

```diff
@@ -158,6 +158,7 @@
         </Key>
         <Key ID="STR_CBA_UI_AdjustingGroupOrder">
             <English>Adjusting group order...</English>
+            <German>Reihenfolge der Gruppen wird angepasst...</German>
```

Comment:
There needs to be space between the last word and the horizontal ellipsis (Auslassungszeichen)

Reply:
Is this for English too?

Reply:
I don’t know